src/modules/exchange/api.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+import type { FixedIo } from '../../io/fixIo.js'
+
+import ENV from '../../../env'
```

Comment:
The core is supposed to be configuration-free. We accept all our API keys and stuff through the `makeContext` call. We can fix this later, since we're in a hurry, but it is a problem.

Reply:
Actually, by the time we get this fixed to work on our build server, it would have been faster to just fix it in the first place. We need to get this taken care of.